src/iqm/iqm_client/iqm_client.py

```diff
@@ -341,6 +343,55 @@ def _validate_instruction(
                 else f'{instruction.qubits} not allowed as locus for {instruction.name}'
             )
 
+    @staticmethod
+    def _validate_circuit_moves(
+        architecture: QuantumArchitectureSpecification, circuit: Circuit, qubit_mapping: Optional[dict[str, str]] = None
+    ) -> None:
+        """Validates that the MOVE gates in the circuit are not exciting the resonator.
+
+        Args:
+          architecture: Quantum architecture to check against.
+          circuit: Quantum circuit to validate.
+          qubit_mapping: Mapping of logical qubit names to physical qubit names.
+              Can be set to ``None`` if the ``circuit`` already uses physical qubit names.
```

Comment:
```suggestion
            architecture: Quantum architecture to check against.
            circuit: Quantum circuit to validate.
            qubit_mapping: Mapping of logical qubit names to physical qubit names.
                Can be set to ``None`` if the ``circuit`` already uses physical qubit names.
```
Not sure it it actually matters but 4 spaces is the standard indentation in docstrings.